rest/server_context.go

```diff
@@ -307,7 +308,7 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		dbName, bucketName, pool, server)
 
 	if err := db.ValidateDatabaseName(dbName); err != nil {
-		return nil, fmt.Errorf("Error validating database name: %v", err)
+		return nil, pkgerrors.Wrapf(err, "Error validating database name")
```

Comment:
All of these errors in getOrAddDatabaseFromConfig feel like they should be wrapped at a lower level (if at all).  Generally these are errors created by SG code, and so could be created as pkgerrors in the first place if we think it's merited.

Reply:
Removed